client/landing/stepper/index.tsx

```diff
@@ -81,6 +82,9 @@ const initializeHotJar = ( flowName: string ) => {
 };
 
 window.AppBoot = async () => {
+	// Start tracking performance, bearing in mind this is a full page load.
+	startStepperPerformanceTracking( { fullPageLoad: true } );
```

Comment:
I suggest moving the `startStepperPerformanceTracking` to after the redirect logic because in theory if the code execute the if condition, we should not consider as a valid flow loading.

```
if ( ! flowName ) {
		// Stop the boot process if we can't determine the flow, reducing the number of edge cases
		return ( window.location.href = `/setup/${ DEFAULT_FLOW }${ window.location.search }` );
	}

startStepperPerformanceTracking( { fullPageLoad: true } );

```

Reply:
Thank you, done! 👍